src/components/Item/Item.js

```diff
@@ -16,18 +16,47 @@ const Item = ({
     type,
     isPlaceholder,
     forwardedPluginRef,
-    onMountChange,
+    onItemRender,
+    onItemMountChange,
+    item,
 }) => {
+    const _isAsyncItem = React.useRef(false);
```

Comment:
Let's not use underscored names for variables as they are already "private"
And maybe better to name it `isAsyncItemRef`